### PR TITLE
[RAPTOR-3238] Allow docker flag to take a directory

### PR DESCRIPTION
**Pull request by 4dahalibut**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary
Users can use this so that they don't need to figure out the python virtualenv stuff if they don't want to.

### CODE REVIEW

File: custom_model_runner/datarobot_drum/drum/drum.py [hunk 1]
```diff
@@ -525,6 +528,28 @@ def _run_inside_docker(self, options, run_mode):
...
+        self.logger.info("This may take some time")
+        try:
+            image, _ = client.images.build(path=docker_image_or_directory)
+        except docker.errors.BuildError as e:
+            self.logger.error("Hey something went wrong with image build!")
+            for line in e.build_log:
+                if "stream" in line:
+                    self.logger.error(line["stream"].strip())
+            raise
+        self.logger.info("Done building image!")
```
Comment (yakov-g) [inline custom_model_runner/datarobot_drum/drum/drum.py:550]: Please update:
1. Changelog
2. Documentation, here: https://github.com/datarobot/datarobot-user-models/tree/master/custom_model_runner#running-inside-a-docker-container

**Comment by 4dahalibut** [comment]
jarvis test functional_drop_in

**Comment by 4dahalibut** [comment]
jarvis test cmrunner_integration

**Comment by 4dahalibut** [comment]
jarvis test functional_drop_in

**Comment by 4dahalibut** [comment]
jarvis test functional_drop_in

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**4.88** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**